drivers/target/iscsi/iscsi_target_nego.c

```diff
@@ -479,12 +479,18 @@ static int iscsi_target_do_login(struct iscsi_conn *, struct iscsi_login *);
 
 static bool __iscsi_target_sk_check_close(struct sock *sk)
 {
-	if (sk->sk_state == TCP_CLOSE_WAIT || sk->sk_state == TCP_CLOSE) {
-		pr_debug("__iscsi_target_sk_check_close: TCP_CLOSE_WAIT|TCP_CLOSE,"
-			"returning FALSE\n");
```

Comment:
```c
pr_debug("... returning FALSE\n");
return true;
```
😭

Reply:
Linux is High Quality Professional Software.